base/tuple.jl

```diff
@@ -20,7 +20,7 @@ length(@nospecialize t::Tuple) = nfields(t)
 firstindex(@nospecialize t::Tuple) = 1
 lastindex(@nospecialize t::Tuple) = length(t)
 size(@nospecialize(t::Tuple), d) = (d == 1) ? length(t) : throw(ArgumentError("invalid tuple dimension $d"))
-axes(@nospecialize t::Tuple) = OneTo(length(t))
+axes(@nospecialize t::Tuple) = (OneTo(length(t)),)
```

Comment:
Yikes. Looks like this needs a test.

Reply:
Wow. Sorry about that.

Reply:
I also deleted the method in Broadcast that was hiding this error